Docs/user_guide/modules/brainsfit.md

```diff
@@ -1,5 +1,194 @@
-```{include} ../../_moduledescriptions/BRAINSFitOverview.md

```

Comment:
Nitpick: Move update to existing module doc and addition of new one into its own commit